skfda/representation/basis.py

```diff
@@ -324,6 +324,42 @@ def _to_R(self):
     def inner_product(self, other):
         return numpy.transpose(other.inner_product(self.to_basis()))
 
+    def add_same_basis(self, coefs1, coefs2):
+        return self.copy(), coefs1 + coefs2
+
+    def add_costant(self, coefs, other):
+        coefs = coefs.copy()
+        other = numpy.array(other)
+        try:
+            coefs[:, 0] = coefs[:, 0] + other
+        except TypeError:
+            raise NotImplementedError
```

Comment:
This should be converted to `return NotImplemented` in the FDataBasis operations. For efficiency, maybe you should return `NotImplemented` here to.

Reply:
This has not been addressed.